### PR TITLE
[zh-cn] Update Getting_started_with_HTML

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

polish the language and translate image alts

### CODE REVIEW

File: files/zh-cn/learn/html/introduction_to_html/getting_started/index.md [hunk 1]
```diff
@@ -30,399 +30,461 @@ slug: Learn/HTML/Introduction_to_HTML/Getting_started
...
-你需要确保元素被正确的嵌套：在上面的例子中我们先打开{{htmlelement("p")}}元素，然后才打开{{htmlelement("strong")}}元素，因此必须先将{{htmlelement("strong")}}元素关闭，然后再去关闭{{htmlelement("p")}}元素。下面的例子是错误的：
+你需要确保元素被正确的嵌套：在上面的例子中我们先打开 `p` 元素，然后才打开 `strong` 元素，因此必须先将 `strong` 元素关闭，然后再去关闭 `p` 元素。下面的例子是错误的：
 
-```html example-bad
+```html-nolint example-bad
 <p>My cat is <strong>very grumpy.</p></strong>
 ```
 
-所有的元素都需要正确的打开和关闭，这样才能按你所想的方式展现。如果像上述的例子一样进行了错误的嵌套，那么浏览器会去猜测你想要表达的意思，但很有可能会得出错误的结果。所以永远不要这么做！
+**所有的元素都需要正确的打开和关闭，这样才能按你所想的方式展现。** 由于上述示例中的那种重叠，浏览器不得不猜测你的意图。这种猜测可能会导致意想不到的结果。
```
Comment (jasonren0403) [inline files/zh-cn/learn/html/introduction_to_html/getting_started/index.md:203]:
```suggestion
**所有的元素都需要正确的打开和关闭，这样才能按你所想的方式展现**。由于上述示例中的那种重叠，浏览器不得不猜测你的意图。这种猜测可能会导致意想不到的结果。
```

File: files/zh-cn/learn/html/introduction_to_html/getting_started/index.md [hunk 2]
```diff
@@ -30,399 +30,461 @@ slug: Learn/HTML/Introduction_to_HTML/Getting_started
...
+<img src="https://roy-tian.github.io/learning-area/extras/getting-started-web/beginner-html-site/images/firefox-icon.png" alt="Firefox 图标">
 ```
 
 显示如下：
 
-{{ EmbedLiveSample('空元素', 700, 300, "", "", "hide-codepen-jsfiddle") }}
+{{ EmbedLiveSample('空元素', 700, 300, "", "") }}
 
-> **备注：** 空元素（Empty elements）有时也被叫作 _void elements_.
+> **备注：** HTML 中，无需在一个无效元素的标签末尾添加 `/`，例如：`<img src="images/cat.jpg" alt="cat" />`。然而，这也是一种有效的语法，当你希望你的 HTML 是有效的 XML 时，这么做也没问题。
```
Comment (jasonren0403) [inline files/zh-cn/learn/html/introduction_to_html/getting_started/index.md:242]:
```suggestion
> **备注：** HTML 中，无需在一个空元素的标签末尾添加 `/`，例如：`<img src="images/cat.jpg" alt="cat" />`。然而，这也是一种有效的语法，当你希望你的 HTML 是有效的 XML 时，这么做也没问题。
```

Comment (jasonren0403) [reply files/zh-cn/learn/html/introduction_to_html/getting_started/index.md:242]:
```suggestion
> **备注：** HTML 中，无需在一个空元素的标签末尾添加 `/`，例如 `<img src="images/cat.jpg" alt="cat" />`。然而，这也是一种有效的语法，当你希望你的 HTML 是有效的 XML 时，这么做也没问题。
```

File: files/zh-cn/learn/html/introduction_to_html/getting_started/index.md [hunk 3]
```diff
@@ -30,399 +30,461 @@ slug: Learn/HTML/Introduction_to_HTML/Getting_started
...
 ## 属性
 
-元素也可以拥有属性，如下：
+元素也可以拥有属性，属性看起来像这样：
 
-![&amp;lt;p class="editor-note">我的猫咪脾气爆&amp;lt;/p>](grumpy-cat-attribute-small.png)
+![含有‘class="editor-note"’属性的段落标签](grumpy-cat-attribute-small.png)
 
-属性包含元素的额外信息，这些信息不会出现在实际的内容中。在上述例子中，这个 class 属性给元素赋了一个识别的名字（id），这个名字此后可以被用来识别此元素的样式信息和其他信息。
+属性包含元素的额外信息，这些信息不会出现在实际的内容中。在上述例子中，这个 **`class`** 属性是一个识别名称，用来为元素提供样式信息。
```
Comment (jasonren0403) [inline files/zh-cn/learn/html/introduction_to_html/getting_started/index.md:250]:
```suggestion
属性包含元素的额外信息，这些信息不会出现在实际的内容中。在上述例子中，这个 **`class`** 属性是一个识别名称，以后为元素设置样式信息时更加方便。
```

File: files/zh-cn/learn/html/introduction_to_html/getting_started/index.md [hunk 4]
```diff
@@ -30,399 +30,461 @@ slug: Learn/HTML/Introduction_to_HTML/Getting_started
...
-3. 一个属性值，由一对引号 ("") 引起来。
+- 一个空格，它在属性和元素名称之间。如果一个元素具有多个属性，则每个属性之间必须由空格分隔。
+- 属性名称，后面跟着一个等于号。
+- 一个属性值，由一对引号（""）引起来。
 
-### 学习实践：为一个元素添加属性
+### 主动学习：为元素添加属性
 
-另一个例子是关于元素{{htmlelement("a")}}的——元素{{htmlelement("a")}}是锚，它使被标签包裹的内容成为一个超链接。此元素也可以添加大量的属性，其中几个如下：
+另一个例子是关于元素 {{htmlelement("a")}} 的——元素 {{htmlelement("a")}} 是*锚*，它使被标签包裹的内容成为一个超链接。锚点可以添加多种属性，部分如下：
```
Comment (jasonren0403) [inline files/zh-cn/learn/html/introduction_to_html/getting_started/index.md:260]:
```suggestion
另一个例子是关于元素 {{htmlelement("a")}} 的——元素 {{htmlelement("a")}} 是*锚*，它使被标签包裹的内容成为一个超链接。锚元素可以添加多种属性，部分如下：
```

File: files/zh-cn/learn/html/introduction_to_html/getting_started/index.md [hunk 5]
```diff
@@ -30,399 +30,461 @@ slug: Learn/HTML/Introduction_to_HTML/Getting_started
...
+
+```html-nolint example-bad
+<a href='https://www.example.com' title='Isn't this fun?'>示例站点链接</a>
 ```
 
-如果你想将引号当作文本显示在 html 中，你就必须使用[实体引用](#实体引用：在_html_中包含特殊字符)。
+要这样做：
+
+```html
+<a href='https://www.example.com' title='Isn&apos;t this fun?'>A link to my example.</a>
```
Comment (jasonren0403) [inline files/zh-cn/learn/html/introduction_to_html/getting_started/index.md:476]:
```suggestion
<a href='https://www.example.com' title='Isn&apos;t this fun?'>示例站点链接</a>
```

File: files/zh-cn/learn/html/introduction_to_html/getting_started/index.md [hunk 6]
```diff
@@ -431,151 +493,179 @@ My cat is very grumpy
...
-    ```html
-    <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
-    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-    ```
+   ```html
+   <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+   ```
 
-    然而这种写法已经过时了，这些内容已成为历史。只需要知道 `<!DOCTYPE html>` 是最短有效的文档声明。
+   文档类型是一个历史遗留问题，需要包含它才能使其他东西正常工作。现在，只需要知道 `<!DOCTYPE html>` 是最短有效的文档声明！
```
Comment (jasonren0403) [inline files/zh-cn/learn/html/introduction_to_html/getting_started/index.md:504]:
```suggestion
   文档类型是一个历史遗留问题，需要包含它才能使其他东西正常工作。现在，只需要知道 `<!DOCTYPE html>` 是最短的有效文档声明！
```

File: files/zh-cn/learn/html/introduction_to_html/getting_started/index.md [hunk 7]
```diff
@@ -638,8 +730,14 @@ My cat is very grumpy
 
 ## 总结
 
-你已经来到了这篇文章的结尾——希望你享受基础的 HTML 学习的旅程。在这里你应该可以理解 HTML 语言的全貌、它基本的工作原理。你应该还学会了一些元素和属性的使用。在这个模块的后续文章中，我们会深入一些你已经见过的东西的细节，并且介绍一些新的 HTML 的特性。未完待续！
+你已经来到了这篇文章的结尾——希望你享受基础的 HTML 学习的旅程。
+
+在这里你应该可以理解 HTML 语言的全貌、它基本的工作原理。你应该还学会了一些元素和属性的使用。在这个模块的后续文章中，我们会深入一些你已经见过的东西的细节，并且展示语言的一些其它概念。
```
Comment (jasonren0403) [inline files/zh-cn/learn/html/introduction_to_html/getting_started/index.md:735]:
```suggestion
在这里你应该可以理解 HTML 语言的全貌和基本的工作原理。你应该还学会了一些元素和属性的使用。在这个模块的后续文章中，我们会深入一些你已经见过的东西的细节，并且展示语言的一些其它概念。
```

File: files/zh-cn/learn/html/introduction_to_html/getting_started/index.md [hunk 8]
```diff
@@ -638,8 +730,14 @@ My cat is very grumpy
...
+在这里你应该可以理解 HTML 语言的全貌和基本的工作原理。你应该还学会了一些元素和属性的使用。在这个模块的后续文章中，我们会深入一些你已经见过的东西的细节，并且展示语言的一些其它概念。
+
+> **备注：** 当你开始学习更多的 HTML 知识时，可能也想了解一些层叠样式列表（[CSS](/zh-CN/docs/Learn/CSS)）的基础知识。CSS 是一种用来设计网页样式的语言（比如，用它改变字体、颜色或页面布局等）。你很快就会发现，HTML 和 CSS 能很好地协调配合。
+
+## 参见
 
-> **备注：** 现在，你将开始学习更多关于 HTML 的知识，你可能也想了解一些层叠样式列表（[CSS](/zh-CN/docs/Learn/CSS)）的基础知识。CSS 是一种用来设计网页样式的语言（比如，用它改变字体、颜色或页面布局等）。你很快就会发现，HTML 和 CSS 能很好地协调配合。
+- [使用 CSS 为 HTML 元素应用颜色](/zh-CN/docs/Web/CSS/CSS_Colors/Applying_color)
 
 {{NextMenu("Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML", "Learn/HTML/Introduction_to_HTML")}}
```
Comment (jasonren0403) [inline files/zh-cn/learn/html/introduction_to_html/getting_started/index.md:743]: 隔壁 mdn/content#24598 准备移除 In this module，所以那一段就不同步过来了

**Review by yin1999 — APPROVED** [review]
:+1:

**Pull request merged**